modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1678,7 +1678,7 @@
                 "format": "java",
                 "type": "Object",
                 "annotations": {
-                  "typeId": "org.mule.runtime.core.routing.AggregationStrategy",
+                  "typeId": "org.mule.runtime.core.internal.routing.AggregationStrategy",
```

Comment:
can internal classes be referenced from the extension model? check with @gsfernandes